pom.xml

```diff
@@ -12,7 +12,7 @@
 
     <groupId>com.fusiondb.studio</groupId>
     <artifactId>fusion-studio-api</artifactId>
-    <version>0.1.2-SNAPSHOT</version>
+    <version>0.1.3-SNAPSHOT</version>
```

Comment:
@ccheraa You don't need to change this manually.

With Maven `-SNAPSHOT` is a living version, where each build has a timestamp.
This version number only gets bumped when we cut an official release.